lib/VolSurface/Calibration/Equities.pm

```diff
@@ -0,0 +1,637 @@
+    my @atmvols            = map { sqrt($_) } @variance;
+    my $calibrated_surface = $self->get_calibrated_surface(\%params);
+    my $actual_surface     = $self->surface;
+
+    my $total = 0;
+    foreach my $day (@{$self->term_by_day}) {
+        my $sum       = 0;
+        my $tenorvega = 0;
+        my $atm_vol   = shift @atmvols;
+        foreach my $point (@{$self->smile_points}) {
+            my $atm_check      = log($point / 100) / $atm_vol / sqrt($day / 365);
+            my $calibrated_vol = $calibrated_surface->{$day}->{$point};
+            my $actual_vol     = $actual_surface->{$day}->{smile}->{$point};
+            my $d1             = (log(100 / $point) + (0.5 * $actual_vol**2) * ($day / 365)) / ($actual_vol * ($day / 365)**0.5);
+            my $nd1            = (1 / (2 * 3.1416)**0.5) * exp(-$d1 * $d1 / 2);
+            my $vega           = $nd1 * ($day / 365)**0.5;
+            $sum += $vega * abs($calibrated_vol - $actual_vol);
+            $tenorvega += $vega;
+        }
+        $total += $sum / $tenorvega;
+    }
+
+    return $total;
+}
+
+=head2 calibration_param_names
+
+Calibration parameter names.
+It is hard-coded here because it needs to be in this sequence
+
+=cut
+
+has calibration_param_names => (
+    is      => 'ro',
+    isa     => 'ArrayRef',
+    default => sub {
+        [qw(
+                atmvolshort
+                atmvol1year
+                atmvolLong
+                atmWingL
+                atmWingR
+                skewshort
+                skew1year
+                skewlong
+                skewwingL
+                skewwingR
+                kurtosisshort
+                kurtosislong
+                kurtosisgrowth
+                )];
+    },
+);
+
+=head2 default_initial_guess
+
+Initial guess for parameters. We need to start with something.
+
+=cut
+
+has default_initial_guess => (
+    is      => 'ro',
+    isa     => 'HashRef',
+    default => sub {
+        {
+            atmvolshort    => 0.13,
+            atmvol1year    => 0.19,
+            atmvolLong     => 0.12,
+            atmWingL       => 1.27,
+            atmWingR       => 1.56,
+            skewshort      => -0.27,
+            skew1year      => -0.08,
+            skewlong       => -0.03,
+            skewwingL      => 2.01,
+            skewwingR      => 2.03,
+            kurtosisshort  => 0.001,
+            kurtosislong   => 0.001,
+            kurtosisgrowth => 0.001
+        };
+    },
+);
+
+=head2 strike_ratio
+
+=cut
+
+has strike_ratio => (
+    is      => 'ro',
+    isa     => 'HashRef',
+    default => sub {
+        {
+            lower  => 0.60,
+            higher => 1.30,
+        };
+    },
+);
+
+=head2 compute_parameterization
+
+    Returns a hash reference with new parameterization and calibration_error.
+    These new parameters are calculated using the current parameterization values saved in cache if present,
+    else it would use default parameterization.
```

Comment:
In POD, a line starting with a space character is example code. Since this is definitely not code, it should not be indented.